.gitlab-ci.yml

```diff
@@ -19,6 +19,10 @@ Lint:
   rules:
     - when: never
 
+K8S Test:
+  rules:
+    - when: never
+
```

Comment:
Are we sure to remove what help use preventing regressions on deployments ?

Reply:
Je ne mesure pas bien ce que fait cette PR, mais comme je disais, ce n'était pas déjà pris en compte dans la PR déjà mergée #125 ? @douglasduteil @gary-van-woerkens